Assets/Scripts/GameController.cs

```diff
@@ -1,10 +1,185 @@
 ﻿using UnityEngine;
+using System;
+using System.Collections.Generic;
+
 
 public class GameController : MonoBehaviour {
 
+	private Token[][] _tokens;
+	private int _gridHeight;
+	private int _gridLength;
+
+	private List<Unit> _units;
```

Comment:
What units? All units? Current team units? Should have a comment or better name.